regl-camera.js

```diff
@@ -27,6 +27,7 @@ function createCamera (regl, props_) {
     dphi: 0
   }
 
+  var element = props.element
```

Comment:
As noted elsewhere, do you think there's a use-case where element would ever be anything but identically the same as the regl context element? If possible, I think it'd be nice to avoid needing to pass this separately if the only information it could add is a potential error that you didn't pass the correct element.